docs/transactions.md

```diff
@@ -1149,305 +1083,213 @@ Deletes a PKI Revocation distribution endpoint (such as RFC5280 Certificate Revo
+- CLI command:
+  - `dcld tx pki add-x509-cert --certificate=<string-or-path> --from=<account>`
 - Validation:
-  - the provided certificate must be a non-root certificate:
+  - provided certificate must not be root:
     - `Issuer` != `Subject`
     - `Authority Key Identifier` != `Subject Key Identifier`
-  - the root certificate must be a NOC certificate and added by the same vendor
-    - `isNoc` field of the root certificate must be set to true
-    - `VID of root certificate` == `VID of account`
   - no existing certificate with the same `<Certificate's Issuer>:<Certificate's Serial Number>` combination.
   - if certificates with the same `<Certificate's Subject>:<Certificate's Subject Key ID>` combination already exist:
-    - the existing certificate must be NOC non-root certificate
-    - the sender's VID must match the vid field of the existing certificates.
-  - the signature (self-signature) and expiration date must be valid.
-- Parameters:
-  - cert: `string` - The NOC non-root Certificate, encoded in X.509v3 PEM format. Can be a PEM string or a file path.
-  - schemaVersion: `optional(uint16)` - Schema version to support backward/forward compatability(default 0)
-- In State:
-  - `pki/ApprovedCertificates/value/<Subject>/<SubjectKeyID>`
-  - `pki/ApprovedCertificatesBySubject/value/<Subject>`
-  - `pki/ApprovedCertificatesBySubjectKeyID/value/<SubjectKeyID>`
-  - `pki/NOCCertificates/value/<VID>`
-  - `pki/ChildCertificates/value/<Certificate's Subject>/<Certificate's Subject Key ID>`
-- CLI Command:
-  - `dcld tx pki add-noc-x509-cert --certificate=<string-or-path> --from=<account>`
+    - the existing certificate must not be NOC certificate.
+    - the sender's VID must match the VID of the existing certificate's owner.
+  - the signature and expiration date are valid.
+  - parent certificate must be already stored on the ledger and a valid chain to some root certificate can be built.
+  - if the parent root certificate is VID scoped:
+    - the provided certificate must also be VID scoped.
+    - the `vid` in the subject of the root certificate must be equal to the `vid` in the subject of the provided certificate.
+    - the `vid` in the subjects of both certificates must be equal to the sender Vendor account's VID.
+  - if the parent root certificate is not VID scoped but has an associated VID:
+    - the provided certificate can be either VID scoped or non-VID scoped.
+    - if the provided certificate is VID scoped, the `vid` in the subject of the certificate must be equal to the VID associated with the root certificate and to the sender Vendor account's VID.
+  - if the parent root certificate is non-VID scoped and does not have an associated VID, an error will occur.
 
-### REVOKE_NOC_X509_ROOT_CERT
+> **_Note:_**  Multiple certificates can refer to the same `<Certificate's Subject>:<Certificate's Subject Key ID>` combination.
+
+#### REVOKE_PAI
```

Comment:
Please add a clarification how it's different from REMOVE_PAI

Reply:
Done